specification/confluent/resource-manager/Microsoft.Confluent/preview/2020-03-01-preview/confluent.json

```diff
@@ -571,6 +576,10 @@
         "display": {
           "description": "The object that represents the operation.",
           "$ref": "#/definitions/OperationDisplay"
+        },
+        "isDataAction": {
+          "description": "Indicates whether the operation is a data action",
+          "type": "boolean"
```

Comment:
Is this intended? I don't see this change in the private pr. It's a breaking change even if it's a preview version.
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-changes-that-require-a-version-change

Reply:
Hi @j5lim,  This is added to resolve s360 swagger correctness. in private-repo it was already present.
 https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/confluent/resource-manager/Microsoft.Confluent/stable/2020-03-01/confluent.json#L580

Reply:
I see the changes approved. Thanks.